src/MicroflowTimer/widget/MicroflowTimer.js

```diff
@@ -161,6 +163,16 @@ define([
             }
         },
 
+        _executeEvent: function() {
+            if(this.callEvent === "callMicroflow" && this.microflow) {
+                this._execMf()
+            } else if (this.callEvent === "callNanoflow" && this.nanoflow){
```

Comment:
DANGER! never just test `this.nanoflow` this will always return true, as it is an object ALWAYS check `this.nanoflow.nanoflow`

Reply:
done